accesshandler/pkg/providers/aws/flask/setup/1.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Locating your ECS Resources
+configFields:
+  - ecsClusterARN
+  - instanceArn
+  - identityStoreId
+---
+
+# Locating your ECS Cluster
+
+Before running anything you'll want to assume an aws session for the account that you are granting access too i.e. `assume aws-production`
+
+We'll be running a few commands with the `aws` CLI - entering their outputs into the fields on screen.
+
+Locate your chosen `ecsClusterARN` by running the following
+```bash
+> aws ecs list-clusters 
+{
+    "clusterArns": [
+        "arn:aws:ecs:ap-southeast-2:{ACN}:cluster/{ID1}",
+        "arn:aws:ecs:ap-southeast-2:{ACN}:cluster/{ID2}",
+        "arn:aws:ecs:ap-southeast-2:{ACN}:cluster/{ID3}",
+    ]
+}
+```
+
+Locate your `InstanceArn` and `IdentityStoreId` via
+```bash
+> aws sso-admin list-instances
+{
+    "Instances": [
+        {
+            "InstanceArn": "arn:aws:sso:::instance/ssoins-825968feece9a0b6",
+            "IdentityStoreId": "d-976708da7d"
+        }
+    ]
+}
```

Comment:
These point to our actual sandbox org - better to replace them with values which are more obviously placeholders.

```suggestion
❯ aws sso-admin list-instances
{
    "Instances": [
        {
            "InstanceArn": "arn:aws:sso:::instance/ssoins-1234567890",
            "IdentityStoreId": "d-1234567890"
        }
    ]
}
```